scripts/lib/api/processHtml.ts

```diff
@@ -353,7 +362,15 @@ export function processMembersAndSetMeta(
 
         if (apiType === "function") {
```

Comment:
This would also be great to test. I know we don't have any tests for this function overall and that it's a bit tricky to test - I hadn't gotten to it because it was more complex hah. 

1. Totally okay to only test `function` and not worry about the others
2. If the overall function is too hard to test, it might be easier to extract a helper function and test that.

Reply:
Good call, I've only added tests for `function` in this PR to keep it focused, but I can follow up with a PR to add others.